File: src/dataset/go/darts_Exercism_go_drapala/claude/ClaudeSonnet4_darts_v4.go
package darts

func Score(x, y float64) int {
	r2 := x*x + y*y

	if r2 <= 1.0 {
		return 10
	}
	if r2 <= 25.0 {
		return 5
	}
	if r2 <= 100.0 {
		return 1
	}
	return 0
}

File: src/dataset/c/darts_exercism-leleah/claude/ClaudeSonnet4_darts_v4.c
#include "darts.h"

uint8_t score (coordinate_t landing_position)
{
    float radius_squared = (landing_position.x * landing_position.x) + (landing_position.y * landing_position.y);
    
    if (radius_squared <= 1.0f) {
        return 10;
    }
    if (radius_squared <= 25.0f) {
        return 5;
    }
    if (radius_squared <= 100.0f) {
        return 1;
    }
    return 0;
}